content/news/2023-05-18-maplibre-newsletter-may-2023/index.md

```diff
@@ -0,0 +1,89 @@
+---
+title: "MapLibre Newsletter May 2023"
+date: 2023-05-16
+categories: ["newsletter"]
+authors: [duje]
+draft: false
+---
+
+# MapLibre Newsletter - May 2023
+
+## Highlights
+
+We would like to thank our first sponsor [Mierune](https://www.mierune.co.jp/) for their continued trust in MapLibre.
```

Comment:
Please show the mierune logo. You can use the same asset as we use on maplibre.org/sponsors.